packages/runtime/src/next/preview-mode.tsx

```diff
@@ -25,6 +25,9 @@ type Props = {
   appOrigin?: string
 }
 
+/**
+ * @deprecated: Use `DraftModeScript` instead.
+ */
```

Comment:
So this is deprecated, but it simply doesn't work anymore, right? Because we're not looking at `x-makeswift-preview-mode` anymore?

I think that if we're going to keep this around we might as well continue using Preview Mode for Pages Router and introduce Draft Mode just for App Router.